crates/kas-widgets/src/menu/submenu.rs

```diff
@@ -148,7 +148,11 @@ impl_scope! {
     impl Widget for Self {
         fn pre_configure(&mut self, mgr: &mut ConfigMgr, id: WidgetId) {
             self.core.id = id;
-            mgr.add_accel_keys(self.id_ref(), self.label.keys());
+            // FIXME: new layer should apply to self.list but not to self.label.
+            // We don't currently have a way to do that. Possibly we should
+            // remove `EventMgr::add_accel_keys` bindings, simply checking all
+            // visible widgets whenever a shortcut key is pressed (also related:
+            // currently all pages of a TabStack have active shortcut keys).
             mgr.new_accel_layer(self.id(), true);
         }
```

Comment:
Note: key handling for menus is somewhat broken due to this. A fix is planned but not prioritised.